server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -26,6 +26,7 @@
 public class CoordinatorDynamicConfig
 {
   public static final String CONFIG_KEY = "coordinator.config";
+  private static final int DEFAULT_LOADING_THROTTLE_LIMIT = -1;
```

Comment:
if there is no reason to have 0 as a limit, then we can use 0 as the default of no limit?